pull-request-template.md

```diff
@@ -0,0 +1,31 @@
+#### What does this PR do?
```

Comment:
I would put this file in the .github directory just because it keeps all the github-related files together in the repo structure.